content/en/examples/admin/konnectivity/egress-selector-configuration.yaml

```diff
@@ -18,4 +18,11 @@ egressSelections:
       # The other supported transport is "tcp". You will need to set up TLS 
       # config to secure the TCP transport.
       uds:
-        udsName: /etc/srv/kubernetes/konnectivity-server/konnectivity-server.socket
+        udsName: /etc/kubernetes/konnectivity-server/konnectivity-server.socket
```

Comment:
Shouldn't socket be placed into `/var/lib/kubernetes/konnectivity-server/konnectivity-server.socket` instead?

Reply:
Why is /var/lib better?

Reply:
Usually `/run` is more common for placing the unix-sockets, but kubernetes, eg. kubelet and CSI already uses `/var/lib` on this purpose:

```console
# find /var/lib/kubelet/ -type s
/var/lib/kubelet/plugins_registry/linstor.csi.linbit.com-reg.sock
/var/lib/kubelet/plugins/linstor.csi.linbit.com/csi.sock
/var/lib/kubelet/pod-resources/kubelet.sock
/var/lib/kubelet/device-plugins/kubelet.sock
```

Reply:
Got it. The advantage of the current layout is that all konnectivity-related configurations are centrally placed under `/etc/kubernetes/konnectivity-server`. I think this advantage surpasses the disadvantage of not following the convention set by kubelet & CSI.

Reply:
OK, let it be!